Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,72 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            bool isValidAge;
+            Console.WriteLine(message);
+            string ageString = Console.ReadLine();
+            isValidAge = string.IsNullOrEmpty(ageString);
+            if (isValidAge) return 0;
```

Comment:
Method name says to PromptInt and your variables talk about age. It shouldn't be so specific, it should talk about int, number or similar abstract target.